roles/lib_openshift/src/ansible/oc_adm_registry.py

```diff
@@ -0,0 +1,47 @@
+# pylint: skip-file
+# flake8: noqa
+
+def main():
+    '''
+    ansible oc module for registry
+    '''
+
+    module = AnsibleModule(
```

Comment:
This looks to be missing ``--credentials=`` which is used in ``roles/openshift_hosted/tasks/registry/registry.yml`` when ``not openshift.common.version_gte_3_2_or_1_2``.